### PR TITLE
Dist/mint 18 jun jul 2022 deduct

### PR DESCRIPTION
**Pull request by luxumbra**
Updating after last mint + a readme update.

I notice that I failed to comment out line 89 in `seed-minting-disburse.js` last time when i ran the dist. I have cross referenced ledger updates across several mints and with this one and to my eye there doesn't seem to be an issue. Cross checking SEED balances from mint 17 to 18 check out also.

I also just realised that I called June - July as mint 18 where it was in fact two mints and should have been 18_19 as we did with April/May (16 & 17). D'you reckon it would be fine to just change the filenames & references?

### CODE REVIEW

**Comment by alalonde** [comment]
> I also just realised that I called June - July as mint 18 where it was in fact two mints and should have been 18_19 as we did with April/May (16 & 17). D'you reckon it would be fine to just change the filenames & references?

It was only a single mint, but covering two months, right?

**Comment by luxumbra** [comment]
Aye, single mint covering 2 months.

**Comment by luxumbra** [comment]
> It was only a single mint, but covering two months, right?

Yes, so you're saying we leave it as is?

File: SeedDistroGuide.md [hunk 1]
```diff
@@ -3,7 +3,13 @@
 ## A. Calculating distribution + updating ledger
 1. Fork the repo locally and follow the instructions in the main readme to load the SourceCred instance (you will need API keys for Github and Discord)
 2. If you already forked the repo, make sure to pull any changes from the master branch (since the discord bot updates ETH addresses in the ledger programmatically)
-3. Run `yarn load` to load the latest contribution data.
+3. Due to painfully long and unreliable method of loading contribution data locally using `yarn load`, we now use the following method from within the root of the XP repo, [as advised by SC](https://discord.com/channels/453243919774253079/718263631158050896/778428725570174986):
```
Comment (alalonde) [inline SeedDistroGuide.md:6]: Nice, yeah, I experimented with something similar to this at some point. Nice to have it written down somewhere!

**Comment by alalonde** [comment]
> > It was only a single mint, but covering two months, right?
> 
> Yes, so you're saying we leave it as is?

Yeah, seems like the mint should correspond to actual mints, not months

**Review by alalonde — COMMENTED** [review]
ship it

**Pull request merged**